paddle/fluid/framework/attribute.h

```diff
@@ -235,7 +268,7 @@ class TypedAttrChecker {
     return *this;
   }
 
-  void operator()(AttributeMap& attr_map) const {
+  void operator()(AttributeMap& attr_map) const {  // NOLINT
```

Comment:
const ？

Reply:
the attr_map is been changed in operator().